runtimes/protocol/identity-management.ts

```diff
@@ -169,6 +175,7 @@ export interface SsoToken {
 
 export interface GetSsoTokenResult {
     ssoToken: SsoToken
```

Comment:
Is `updateCredentialsParams` the only data that are actually going to be used and sent through the LSP client back to the Q Server? 
If so why do we need to keep the original `ssoToken`?

Reply:
ssoToken should remain for the original purpose of the destination using the bearer token for local calls (from the destination directly to AWS services).  Today that is CodeCatalyst, but other AWS services are expected to support bearer tokens in the future.  The updateCredentialsParams was just added to make it so destinations didn't have to perform work (beyond just making the call) to execute an Auth update call.